### ISSUE TITLE
Add aurelia-toolbelt prevents bootstrapping the app

### ISSUE THREAD

**Issue by shahabganji** [MaximBalaganskiy/AureliaDotnetTemplate#4]
Hi @MaximBalaganskiy ,

I [forked](https://github.com/shahabganji/AureliaDotnetTemplate) your repository and just added the ```aurelia-toolbelt``` package to it, however, one way or the other it has an error at bootstrapping the Aurelia app.

![image](https://user-images.githubusercontent.com/4405518/46913577-d7f7f700-cf9c-11e8-88ee-9aca0788a5d5.png)

I have just added this line to the `boot.ts` file

```ts
aurelia.use.standardConfiguration()
	.plugin(PLATFORM.moduleName("aurelia-toolbelt"));
```

Can you take a look at this problem, or give me some hints?

**Comment by shahabganji** [comment]
@MaximBalaganskiy 

I speculate that it is related to `aurelia-dialog` too, what I see from the logs of the issue, however, to make sure I'll check with older versions of `aurelia-toolbelt` in which we do not have `aurelia-dialog`.

One question out of my curiosity, how this repository differs from what `aurelia-cli` creates? does it have benefits over cli version? 

Have a great day :+1:

**Comment by MaximBalaganskiy** [comment]
It was created as a continuation of the retired official dotnet template.
Last time I looked, cli handles bundling slightly differently - it requires
a manual list of vendor packages plus there is no bs4 in the cli's template

**Comment by gregoryagu** [comment]
Check out my fork https://github.com/gregoryagu/AureliaDotnetTemplate It's been updated a bit from the original.

**Comment by shahabganji** [comment]
Thanks, @gregoryagu, 

I checked your fork and it has the same problem with `aurelia-dialog`

**Comment by MaximBalaganskiy** [comment]
Is this still relevant?

**Comment by shahabganji** [comment]
I would prefer this template, so the answer is yes, have you changed anything?

**Comment by shahabganji** [comment]
So, the workaround is to turn off the module concatenation plugin? I didn't see that you have used that in this template, right?

**Comment by shahabganji** [comment]
Yes, however, aurelia-dialog plugin is one that will be of use in most of Aurelia based SPAs.

**Comment by MaximBalaganskiy** [comment]
Can you please check that the fix is working for your package?

**Comment by shahabganji** [comment]
> Can you please check that the fix is working for your package?

The fix is to turn off the `module concatenation in webpack`? I'll check.

**Comment by MaximBalaganskiy** [comment]
@shahabganji  Any luck?

**Comment by shahabganji** [comment]
I’ll check it on the weekend.

**Comment by shahabganji** [comment]
@MaximBalaganskiy 

I checked this on my fork and changed some stuff, however, when I cloned your repository again and checked aurelia-dialog and aurelia-toolbelt, both look to work properly. I am not sure why, but maybe sth related to [this commit](https://github.com/MaximBalaganskiy/AureliaDotnetTemplate/commit/c1570b22c76ce49178f8055ab543ac4c5d5a2b90). Anyway I added aurelia-toolbelt on my [forked repository](https://github.com/shahabganji/AureliaDotnetTemplate). If you'd like I can make a pull request, otherwise there might not be any big differences. BTW, I am not sure but I remember that one said MiniCssExtraxtPlugin might be better than ExtractTextWebpackPlugin. What's your opinion of replacing these two guys?

And, since I didn't see the problem any more I'll close the issue.

Thanks
